sdk/ml/azure-ai-ml/tests/pipeline_job/e2etests/test_pipeline_job.py

```diff
@@ -1432,6 +1432,36 @@ def test_register_output_yaml(self, client: MLClient, randstr: Callable[[str], s
         assert node_output.name == 'convert_data_node_output'
         assert node_output.version == '1'
 
+    def test_register_output_yaml_succeed(self, client: MLClient, randstr: Callable[[str], str],):
+        register_pipeline_path = "./tests/test_configs/dsl_pipeline/pipeline_with_pipeline_component/pipeline_register_output.yml"
+        pipeline = load_job(source=register_pipeline_path)
+        # overwrite version
+        random_version = randstr("version")
```

Comment:
will this output a different version for each run?

Reply:
modified here with fixed version